search_engine/search_engine/webcrawler.py

```diff
@@ -11,12 +11,13 @@ class MyHTMLParser(HTMLParser):
     def handle_starttag(self, tag, attrs):
         if tag == 'a':
             attr = dict(attrs)
-            if 'href' in attr:   
+            if 'href' in attr and not attr['href'].startswith('mailto:') \
+                and not attr['href'].startswith('tel:'):
```

Comment:
Coding Standard Violation:
The continuation line should not have the same indent as next logical line.

Reply:
You do realise that's a deleted line, right?

Reply:
I am talking about `and not attr['href'].startswith('tel:'):`

Reply:
![Screenshot from 2019-07-14 14-24-18](https://user-images.githubusercontent.com/22322152/61181450-5f65ba00-a644-11e9-965e-199e3af187b4.png)